ckanext/iati/views/spreadsheet.py

```diff
@@ -226,7 +226,8 @@ def csv_upload_datasets():
         vars['file_name'] = csv_file.filename
         data = io.BytesIO(_data)
```

Comment:
This looks like a complicated version of:

```
reader = csv.DictReader(data)
for row in reader:
     task={}
     task['title'] = row.get('title', '') or "No Title"
     task['task_id'] = str(uuid.uuid4())
     job = jobs.enqueue(records_upload_process,
                               [json.dumps([row], ensure_ascii=False).encode('utf-8'), c.user])
     task['task_id'] = str(job.id)
     tasks.append(json.dumps(task))
```

Except I really don't understand enqueuing an array of [bytes, object], where the bytes is a json string of an array of a single dict. But I guess that's what's required upstream.


Do we have any test files that are hitting these various encodes/decodes?